widget/charts/src/components/BarChart/BarChart.types.ts

```diff
@@ -30,3 +30,27 @@ export type TooltipDataType = {
   };
   hoveredIndex: number;
 };
+
+export type DailyDataType = {
+  date: string;
+  bucket: string;
+  count: number;
+  volume: number;
+  txCountEffective: number;
+  volumeEffective: number;
+};
+
+export type ChartOptionsType =
```

Comment:
Please remove the **`isStackBar`**, add a **`defaultLabel`** to it, and handle it without a stack inside this method.
Also, we can pass the color as a single-member array in the non-stack mode

Reply:
fixed